Dockerfile.release

```diff
@@ -1,23 +1,31 @@
-FROM bitwalker/alpine-erlang:19.2.1b
+FROM phusion/baseimage
```

Comment:
I can still follow up on that. Although the problem seems to be with the Erlang's VM itself. The OS doesn't have any issues with the DNS while `beam` is not able to resolve any domains when the OS is setup to use docker-swarm's integrated DNS.